### PR TITLE
docs(readme.md): 英文文档

### PR DESCRIPTION
**Pull request by gd4Ark**
## Why
目前只有中文文档，需要新增英文文档

## How
原本的 `README.md` 改为 `README-zh.md`，另新增 `README.md` 编写英文文档

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,33 +36,33 @@ yarn add @femessage/update-popup
 
 ## Usage
 
-你需要通过环境变量 `UPDATE_POPUP_VERSION` 来传入版本号，后续每次迭代更新只需要修改比当前大的版本号即可。
+You need to use environment variables `UPDATE_POPUP_VERSION` To pass in the version number, when iteratively updating, modify version number greater than the previous version number
```
Comment (snowlocked) [inline README.md:39]: `You need to set environment variables `UPDATE_POPUP_VERSION` To pass in the version number, when iteratively updating, modify version number greater than the previous version number`

File: README.md [hunk 2]
```diff
@@ -36,33 +36,33 @@ yarn add @femessage/update-popup
...
-  chainWebpack: config => {
+  chainWebpack: (config) => {
     config.plugin('femessage-update-popup').use(UpdatePopup, [{options}])
-  }
+  },
 }
 ```
 
-就这么简单！
+It's that easy.
```
Comment (snowlocked) [inline README.md:65]: `It's so easy.`

**Review by shoyuf — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ const config = {
 - Default: `webpackConfig.output.publicPath`
 - Reference: [webpack publicPath](https://webpack.docschina.org/configuration/output/#outputpublicpath)
 
-使用独立的 publicPath，一般情况下不需要设置此参数。
+Using publicPath, you do not normally need to set this parameter.
```
Comment (snowlocked) [inline README.md:77]: `To use independent publicPath, you need not to set this parameter under normal circumstances.`

**Review by snowlocked — APPROVED** [review]

**Review by shoyuf — APPROVED** [review]

**Comment by levy9527** [comment]
@all-contributors add @gd4Ark doc, translation

**Pull request merged**